piker/data/_ahab.py

```diff
@@ -143,27 +143,37 @@ async def process_logs_until(
                 if not entry:
                     continue
 
+                entry = entry.strip()
                 try:
-                    record = json.loads(entry.strip())
+                    record = json.loads(entry)
+
+                    if 'msg' in record:
+                        msg = record['msg']
+                    elif 'message' in record:
+                        msg = record['message']
+                    else:
+                        raise KeyError('Unexpected log format')
```

Comment:
dump the `msg` contents here in the exception message as well so that if/when this breaks we have a clue what the problematic msg was.